gitpod-io/website
# [Analytics] Track whether user is authenticated in each page and track call

## Description

Includes authentication status of users in props of all `page()` and `track()` calls by parsing the `gitpod-user` cookie. this enables that page visits can more easily be segmented between users and visitors and that third party tools can trigger actions based on authentication status.


## How to test


1. open the gitpod website in the preview environment
2. click on any button on the website
3. go to the [segment debugger of website staging](https://app.segment.com/gitpod/sources/website_staging/debugger) and ensure that the `authenticated` property is passed for both, the initial `page` call and the `track("website_clicked")` call
